test/chpldoc/compflags/version/projectVersion.doc.catfiles

```diff
@@ -0,0 +1,7 @@
+docs/objects.inv                        # projectVersion.*.object.doc.good
+docs/chpl-modindex.html                 # projectVersion.*.modindex.doc.good
+docs/genindex.html                      # projectVersion.*.genindex.doc.good
+docs/index.html                         # projectVersion.*.index.doc.good
+docs/search.html                        # projectVersion.*.search.doc.good
+docs/modules/projectVersion.doc.html    # projectVersion.*.html.doc.good
+docs/_static/documentation_option.js    # projectVersion.*.option.doc.good
```

Comment:
Does specifying a .good in a .catfiles actually work?  I would not expect that to work